net/file_server.ts

```diff
@@ -58,202 +24,9 @@ if (target) {
   currentDir = `${currentDir}/${target}`;
 }
 const addr = `0.0.0.0:${serverArgs[2] || 4500}`;
-const encoder = new TextEncoder();
-
-function modeToString(isDir: boolean, maybeMode: number | null) {
-  const modeMap = ["---", "--x", "-w-", "-wx", "r--", "r-x", "rw-", "rwx"];
-
-  if (maybeMode === null) {
-    return "(unknown mode)";
-  }
-  const mode = maybeMode!.toString(8);
-  if (mode.length < 3) {
-    return "(unknown mode)";
-  }
-  let output = "";
-  mode
-    .split("")
-    .reverse()
-    .slice(0, 3)
-    .forEach(v => {
-      output = modeMap[+v] + output;
-    });
-  output = `(${isDir ? "d" : "-"}${output})`;
-  return output;
-}
-
-function fileLenToString(len: number) {
-  const multipler = 1024;
-  let base = 1;
-  const suffix = ["B", "K", "M", "G", "T"];
-  let suffixIndex = 0;
-
-  while (base * multipler < len) {
-    if (suffixIndex >= suffix.length - 1) {
-      break;
-    }
-    base *= multipler;
-    suffixIndex++;
-  }
-
-  return `${(len / base).toFixed(2)}${suffix[suffixIndex]}`;
-}
 
-function createDirEntryDisplay(
-  name: string,
-  path: string,
-  size: number | null,
-  mode: number | null,
-  isDir: boolean
-) {
-  const sizeStr = size === null ? "" : "" + fileLenToString(size!);
-  return `
-  <tr><td class="mode">${modeToString(
-    isDir,
-    mode
-  )}</td><td>${sizeStr}</td><td><a href="${path}">${name}${
-    isDir ? "/" : ""
-  }</a></td>
-  </tr>
-  `;
+if (fileServer) {
```

Comment:
This condition is not needed, `fileServer` is an import and will always be defined.

Reply:
how can i achieve this kind of feature in deno
```
def fileServer(...):
   ...

if __name__ == '__main__':
    fileServer(...)
```

Reply:
You can't, and it's actually not needed, the code you wrote is what we want 👍